docs/plans/2026-04-30-expanded-view-dollar-context-design.md

```
@@ -0,0 +1,243 @@
+**File layout**
+- `Sources/OpenIslandCore/ContextUsage.swift` — types + parser + table.
+- `Sources/OpenIslandApp/Views/ContextLeftBadge.swift`
+- `Sources/OpenIslandApp/Views/DollarTodayPill.swift`
+- `Sources/OpenIslandApp/AppModel.swift` — registry + headerNeedsCodeburn flag.
+- `Sources/OpenIslandApp/Views/IslandPanelView.swift` — wire both views.
+
+## Data flow
+
+### `$` today pill
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)